content/preregistration.md

```diff
@@ -0,0 +1,27 @@
+# Preregistration
+
+Predictions are tested on data; preregistration helps prove that predictions precede the data. This chronology is essential for valid tests of predictions (confirmatory research). Preregistration is not necessary for research that has no predictions and uses no hypothesis testing (exploratory research). Preregistration derives its name from the act of registering predictions before (pre) a publication finds place (not to be confused with [Preprints](preprints.md)).
```

Comment:
"publication finds place..." -> A little confusing wording. 

Should preregistration happen at the start of a project, before the data collection/analysis and long before publication takes shape? I wonder if this varies across fields, but for astronomy I would assume preregistration would take place around the proposal stage.

Perhaps (although feel free to ignore): "registering predictions before (pre) analysis" ?

Reply:
Thanks, good suggestion! It does vary probably and your revision gets right to the essence.